registry/config.go

```diff
@@ -331,10 +331,6 @@ func (config *ServiceConfig) NewRepositoryInfo(reposName string) (*RepositoryInf
 
 	if repoInfo.Index.Official {
 		normalizedName := repoInfo.RemoteName
-		if strings.HasPrefix(normalizedName, "library/") {
-			// If pull "library/foo", it's stored locally under "foo"
-			normalizedName = strings.SplitN(normalizedName, "/", 2)[1]
```

Comment:
Does it mean that what was previously displayed as `busybox` in the UI might now appear as `library/busybox` or is it completely unrelated?